README.md

```diff
@@ -1091,12 +1091,15 @@ IEEE Int. Conf. Robotics and Automation (ICRA), 2019. [PDF](http://rpg.ifi.uzh.c
 - <a name="Lee19icraw"></a>Lee, A. J., Cho, Y., Yoon, S., Shin, Y., Kim, A.,  
 [ViViD: Vision for Visibility Dataset](https://sites.google.com/view/dgbicra2019-vivid/),  
 IEEE Int. Conf. Robotics and Automation (ICRA) Workshop: Dataset Generation and Benchmarking of SLAM Algorithms for Robotics and VR/AR, 2019.
+- <a name="Mitrokhin19iros"></a>Mitrokhin, A., Ye, C., Fermüller, C., Aloimonos, Y., Delbrück, T.,  
+[EV-IMO: Motion Segmentation Dataset and Learning Pipeline for Event Cameras](https://arxiv.org/abs/1903.07520),  
+IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2019. [PDF](https://arxiv.org/pdf/1903.07520.pdf), [Dataset](https://better-flow.github.io/evimo), [Project page](http://prg.cs.umd.edu/EV-IMO.html)
```

Comment:
Thanks, Anton. To make the repository easier to maintain, if a paper appears in multiple sections, there should be only one primary entry; the others should be links to the primary one. See for example, the paper by 
Mueggler et. al. IJRR 2017. The Event-Camera Dataset and Simulator... 
Please use these guidelines for the EV-IMO paper.

Reply:
BTW, I am planning to create a segmentation section, to collect papers dedicated to that problem. Perhaps it would make more sense to place the paper in this new section.